pkg/csi/cinder/openstack/openstack.go

```diff
@@ -90,13 +90,16 @@ type BlockStorageOpts struct {
 }
 
 type Config struct {
-	Global       client.AuthOpts
+	Global       map[string]*client.AuthOpts
```

Comment:
Why a pointer?

Reply:
Because it's a requirement of gcfg library for subsections handle https://pkg.go.dev/gopkg.in/gcfg.v1#example-ReadStringInto-Subsections
If I use type `map[string]client.AuthOpts` type, I got following error on execution of function `gcfg.FatalOnly(gcfg.ReadInto(&cfg, config))`:
```
panic: map field for section must have string keys and  pointer-to-struct values: section "Global"
```
Do you have a better solution in mind ?